OnlineShopAPI/IdentityServer/Controllers/UserController.cs

```diff
@@ -47,14 +47,14 @@ public async Task<ActionResult<UserDto>> RegisterUser([FromBody] UserRegisterDto
                 return BadRequest(result.ToErrorsString());
             }
 
-            var createdUser = await _uow.UserRepository.GetUserByUsernameAsync(userRegisterDto.UserName);
+            //var createdUser = await _uow.UserRepository.GetUserByUsernameAsync(userRegisterDto.UserName);
```

Comment:
Code in the PR must not contain any redundant comments. Tend to avoid comments in the future